telosnetwork/telos-wallet
# #415 | Back button behavior

# Fixes #415 

## Description
See ticket

## Test scenarios
- go to https://deploy-preview-416--wallet-staging.netlify.app/
- sign in
- go to https://deploy-preview-416--wallet-staging.netlify.app/evm/send
- press the back button in the app header
    - you should be taken to the wallet page
- go to https://deploy-preview-416--wallet-staging.netlify.app/evm/collectible-details?contract=0x0000000000000000000000000000000000000000&id=123456
- press the back button in the app header
    - you should be taken to the inventory

## Checklist:
